apps/admin_app/lib/admin_app_web/helpers.ex

```diff
@@ -122,30 +158,35 @@ defmodule AdminAppWeb.Helpers do
     end
   end
 
-  def order_xlsx_exporter(user) do
-    orders = Repo.all(Order) |> Repo.preload([:line_items])
+  def xlsx_exporter(user, type) do
```

Comment:
@jyotigautam this should definitely be moved out of helpers. @SagarKarwande @pkrawat1 what do you guys think?

Reply:
Please see [here](https://github.com/spree/spree/tree/master/core/lib/spree/core/importer) how it has been organised in the `lib` folder. We need to do like this. I don't think we have something like this as of now.